add-ons/prometheus-3.7/README.md

```diff
@@ -0,0 +1,32 @@
+# Prometheus Add-on
+An addon that will deploy Prometheus, Node-Exporter and AlertManager.
+
+NOTE: Requires Origin >= 3.7.0-rc.0
+
+Verify you have installed these addons, by following the [general readme](../../README.adoc#download-and-use-community-add-ons).
+
+## Deploy prometheus
+To deploy prometheus do:
+
+```
+minishift addon apply prometheus --addon-env prometheus_namespace=kube-system
```

Comment:
Add-on names are decided from the directory names of the add-on, so the name of the add-on for this PR is `prometheus-3.7`.  But the readme says that the name is `prometheus`.  So I am little confused.